.github/CODEOWNERS

```diff
@@ -12,6 +12,7 @@
 #   can be optimized further.
 
 # Angular Migration team
+/core/templates/dev/head/services/contextual/logger.service.ts @bansalnitish
```

Comment:
Any specific reason for placing it over here? (inside `Angular Migration team`)

Reply:
No.